packages/validator/test/unit/utils/interopConfigs.ts

```diff
@@ -17,6 +17,8 @@ export const lighthouseHoleskyConfig = {
   CAPELLA_FORK_EPOCH: "256",
   DENEB_FORK_VERSION: "0x05017000",
   DENEB_FORK_EPOCH: "29696",
+  FULU_FORK_EPOCH: "419696",
```

Comment:
is this actually required?

Reply:
not sure but you mentioned to add to all the configs so I did.... no idea what that file is for actually but i suppose I could do some digging....

Reply:
As in I should remove this??? ok... removed :)

Reply:
it's not critical, what we probably wanna do is pull the actual config that lighthouse returns so the tests make sense, last time I did that was quite a while ago

Reply:
> is this actually required?

so what I really wanted to know is if this is required to pass unit tests but it looks like that's not the case, could have phrased that a bit better